src/osd/PrimaryLogPG.cc

```diff
@@ -3303,6 +3304,34 @@ void PrimaryLogPG::do_proxy_chunked_op(OpRequestRef op, const hobject_t& missing
   } 
 }
 
+void PrimaryLogPG::refcount_manifest(ObjectContextRef obc, object_locator_t oloc, hobject_t soid,
+                                     SnapContext snapc, string method, Context *cb, uint64_t offset)
```

Comment:
s/string method/bool get/ so we can avoid a string comparison here

Reply:
Fixed